docs_src/general/EdgeX_CN.md

```diff
@@ -0,0 +1,153 @@
+
+**Serverless Functions**
+
+EdgeX is not built on a serverless execution model.  Unlike the compute and infrastructure resources of the cloud (which can almost be thought of as infinitely available and scaled up or down as needed), edge compute resources and infrastructure are not scaled up or down based on demand.  An edge gateway, running on a light pole of a smart city for example, is not dynamic.  The gateway must be provisioned based on the expected highest demand of that platform.  The workload on the edge gateway must operate within those resource constraints.  EdgeX is designed to operate in some of the smaller of the static, resource constrained environments.
+
+**Cloud**
+
+Interestingly, we have been asked it EdgeX can run in the cloud.  Indeed, some services (such as application services or analytics packages like the rules engine) could run in the cloud (most of the services are platform agnostic), but EdgeX was designed to serve as the middleware between the edge and the cloud.  At the lowest level - EdgeX services are meant to connect the physical edge (IoT senors and devices of the OT world) to IT worlds.  EdgeX connects things that don't always speak TCP/IP based IT protocols.  EdgeX is meant to explore data at the edge in order to reduce latency of communication (making decisions closer to where the decision is turned into action) with the physical edge and reduce the amount of data that needs to be back-hauled to the world of IT (reducing the transportation and storage of unimportant edge data).  Even if physical sensors or devices are able to connect and talk to the cloud directly (perhaps because they have Wifi or 5G capability allowing them to connect via TCP/IP), the latency needs and cost to transport all the data directly to the cloud is typically prohibitive.
+
+!!! Note
+    There are some edge use case where a sensor-to-cloud architecture is warranted.  Where the sensor speaks well known IT protocols (TCP/IP REST, MQTT, etc.), the edge data collection rates are small, and there is no need to make quick decisions at the edge, a simple sensor to cloud architecture makes sense and would likely negate the need for EdgeX in that situation.
```

Comment:
I am someone unsatisfied with this explanation but can't quite elucidate why.  Bringing in the cloud brings in the requirement for distribution, and as explained earlier, the requirement for distribution brings increased requirements for security, and increased requirements for security increase the footprint of the architecture such that it might not run on constrained devices anymore, which was what EdgeX was targeting in the first place.  Also, as you note, if you have highly capable smart devices, and a highly capable cloud, EdgeX can end up getting squeezed out.

Reply:
happy to rewrite or change if I can get some handle on where it needs to go.  Again - per above, this section of the document is trying to elucidate how/why EdgeX does not conform to the cloud native principals or have CNCF ingredients.  An obvious one is the fact that EdgeX was not design to function or run in the cloud (period full stop).